src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2028,6 +2028,21 @@ Here is the ordered list of rules for the EDF policy:
 
 6. job with earliest submission time.
 
+==== Email notificaions
+
+EDF policy sends 3 types of email notifications:
+
+1. [UNDER CONSTRUCTION] when job deadline is at risk (if we do not start job in next X minutes we will miss its deadline)
```

Comment:
if we do not start job in next X minutes we will miss its deadline --> if the job is not started in the next X minutes, its deadline will be missed